examples/fitness/lib/feed.dart

```diff
@@ -171,28 +171,59 @@ class FeedFragment extends StatefulComponent {
     });
   }
 
+  Widget buildChart() {
+    double startX;
+    double endX;
+    double startY;
+    double endY;
+    List<Point> dataSet = new List<Point>();
+    for (FitnessItem item in userData) {
+      if (item is Measurement) {
+          double x = item.when.millisecondsSinceEpoch.toDouble();
```

Comment:
I suspect we're going to want to show weights quantized to days instead of milliseconds. :)